extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -96,13 +110,45 @@ export class NetCoreTool {
 		return undefined;
 	}
 
+	private get isNetCoreVersionSupported(): boolean | undefined {
+		try {
+			let spawnSync = require('child_process').spawnSync;
+			let child;
+
+			child = spawnSync('dotnet --version', [], {
```

Comment:
You should avoid using sync functions for potentially long-running operations (basically any external operations). Would suggest using the async versions and having this function be async itself.

Reply:
Fixed it to use async version of spawn.